contract_interface.md

```diff
@@ -1,6 +1,6 @@
-# eWASM Contract Interface (ECI) Specification (*Revision 0*)
+# Ewasm Contract Interface (ECI) Specification (*Revision 0*)
 
-The eWASM Contract Interface (ECI) specifies the structure of a contract module.
+The Ewasm Contract Interface (ECI) specifies the structure of a contract module.
```

Comment:
So is it ewasm or Ewasm?

Reply:
As I noted in the PR, I figured that the name should be treated as a common noun but stylized with lowercase. So in titles, acronyms, and at the beginning of sentences I prefer Ewasm but in other cases it is ewasm.